src/pages/index.js

```diff
@@ -194,11 +217,97 @@ const popupWithAddForm = new PopupWithForm(
   "#add-card-modal",
   handleAddCardSubmit
 );
+
 popupWithEditForm.setEventListeners();
 popupWithAddForm.setEventListeners();
 
+const cardSection = new Section(
+  {
+    items: initialCards,
+    renderer: (item) => {
+      const card = createCard(item);
+      cardSection.addItem(card);
+      ``` 1`;
+    },
+  },
+  ".cards__list"
+);
+
+const userData = new UserInfo(profileName, profileDescription, avatar);
+
+api
+  .getInitialCards()
+  .then((cardData) => {
+    const cardSection = new Section(
```

Comment:
Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.